lib/ansible/playbook/role/requirement.py

```diff
@@ -163,7 +163,7 @@ def run_scm_cmd(cmd, tempdir):
         run_scm_cmd(clone_cmd, tempdir)
 
         if scm == 'git' and version:
-            checkout_cmd = [scm_path, 'checkout', version]
+            checkout_cmd = [scm_path, 'checkout', str(version)]
```

Comment:
Please use `to_text()` rather than `str()`:
```suggestion
            checkout_cmd = [scm_path, 'checkout', to_text(version)]
```

This can be imported from `ansible.module_utils._text`

Reply:
Horosho :)))) will do it